view/adminhtml/web/js/sentry-config.js

```diff
@@ -0,0 +1,29 @@
+Sentry.init({
+    dsn: 'https://a832bffb9c574aaa9e4a113b61f90e9e@sentry.io/1263830',
+    beforeSend(event) {
+        var isBreadError = event.fingerprint && event.fingerprint.includes('breadError');
+
+        if (!isBreadError) {
+            return null;
+        }
+        return event;
+    }
+});
+
+var handleError = function(message, extraContext) {
+    Sentry.withScope(function(scope) {
+
+        if (extraContext) {
+            extraContext.forEach(function(context) {
+                scope.setExtra(context.key, context.value);
+            });
+        }
+
+        // add fingerprint to distinguish custom thrown errors from generic errors from outside Bread
+        scope.setFingerprint(['breadError']);
+
+        Sentry.captureMessage(message);
+    });
+
+    console.error(message);
+};
```

Comment:
No need for two identical js files, for each area, you can place it inside **base** folder and it can be loaded from there in both adminhtml and frontend scopes.

Reply:
yeah I sent you a message about that, because I couldn't get the base folder to pull things in properly